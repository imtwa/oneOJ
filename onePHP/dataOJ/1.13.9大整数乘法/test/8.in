5551266837434577575817645338143443117577748832511632727726526327661247784453755741358258846581387654877433884667183481153815171456334384457445154666815341411484461634644673531252526238814423
4735228753245428766225732354328466658626552761786358513856211336745281624681766423777133231434643755263566524712318634777763274233655541454678173357817888717374561188532576412112574143733828