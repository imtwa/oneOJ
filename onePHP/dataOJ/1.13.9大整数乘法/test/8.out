26286518345558029064322143202817084211561771075354031967874476471936914790119396889271790844891598697813529154269252593322757895379963065390840387925430728405921629459227636260161268301066271746524545126562450777705615787390499460409944707367911679536240623676378546713684213340567836353442069993438603172982991685219085731711982638759633492405956025341732881201801830041199401244